multimod/README.md

```diff
@@ -0,0 +1,151 @@
+# Go MultiMod Releaser
+
+This Go Cobra app adds versioning support for repos containing multiple Go
+Modules. Specifically, this app allows for repo maintainers to specify versions
+for sets of Go Modules, so that different sets may be versioned separately.
+
+## Specify Module Sets and Versions
+
+To specify sets of modules whose versions will be incremented in lockstep, the
+`versions.yaml` file must be manually edited between versions. When creating a
+`versions.yaml` file...
+
+* For each module set, give it a name and specify its version and modules it
+  includes.
+* Specify the Go module's import path (rather than its file path)
+* Specify modules which should be excluded from versioning
+* Ensure versions are consistent with the semver versioning requirements.
+* Update version numbers or module groupings as needed for new releases.
+
+An example versioning file is given in [the versions-example.yaml
+file](./docs/versions-example.yaml).
+
+## Creating the app binary
+
+TODO: switch to automatically pulling newest version of `multimod` app binary.
+
+This section describes the current process used for building the Cobra app
+binary, but the process will be updated soon to automatically fetch the most
+recent version of the app.
+
+To build the binary, simply use the following commands:
+
+```sh
+# from the opentelemetry-go-build-tools repo root
+cd multimod
+go build -o multimod main.go
+```
+
+## Verify Module Versioning
+
+Once changes have been completed to the `versions.yaml` file, and the `multimod`
+file is within the target repo, verify that the versioning is correct by running
+the `verify` subcommand:
+
+```sh
+# within the target repo
+./multimod verify
```

Comment:
That is true, but I couldn't find a way to specify the output name of the binary... it seems to always match the package name, which happens to be "package main". As far as I know, this would require me to repackage the Cobra application as a package called "multimod" (rather than "main), and then it could be run using these commands?
```
go install multimod.go
multimod verify
...
```

If that sounds right, I can definitely include that into the next PR for this!